test/common/network/udp_listener_impl_test.cc

```diff
@@ -84,7 +101,11 @@ class UdpListenerImplTest : public ListenerImplTestBase {
     EXPECT_EQ(*data.addresses_.local_, *send_to_addr_);
 
     size_t num_packet_per_recv = 1u;
-    if (Api::OsSysCallsSingleton::get().supportsMmsg()) {
+    if (Api::OsSysCallsSingleton::get().supportsUdpGro()) {
+      // As in UdpGro Test, the 4 packets stacked together
+      // get delivered in a single recvmsg syscall
+      num_packet_per_recv = 4u;
```

Comment:
Is it easier to pass in num_packet_per_recv as a parameter instead? So you can have more control in the tests.

Reply:
Added num_packets_per_recv argument to the validateRecvCallbackParam function. 
```
void validateRecvCallbackParams(const UdpRecvData& data, 
                                size_t num_packet_per_recv)
```